wagtail/wagtailembeds/embeds.py

```diff
@@ -31,7 +33,13 @@ def embedly(url, max_width=None, key=None):
 
     # Get embedly key
     if key is None:
-        key = settings.EMBEDLY_KEY
+        try:
+            key = settings.WAGTAILEMBEDS_EMBEDLY_KEY
+        except AttributeError:
+            warnings.warn(
+                "use WAGTAILEMBEDS_EMBEDLY_KEY instead",
```

Comment:
We should give this message some more context:
"EMBEDLY_KEY is now deprecated. Use WAGTAILEMBEDS_EMBEDLY_KEY instead"